src/test/java/com/thealgorithms/dynamicprogramming/UniqueSubsequencesCountTest.java

```diff
@@ -0,0 +1,55 @@
+package com.thealgorithms.dynamicprogramming;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class UniqueSubsequencesCountTest {
+
+    @Test
+    void subseqCountTestOne() {
+        String s = "abc";
+
+        assertEquals(7, UniqueSubsequencesCount.countSubsequences(s));
+    }
+
+    @Test
+    void subseqCountTestTwo() {

```

Comment:
The code not written via parameterized tests approach. Please you them. There are a lot of such examples in project.